src/scripts/farming/aura/Aura.ts

```diff
@@ -0,0 +1,68 @@
+class Aura {
+
+    auraType: AuraType;
+    auraMultipliers: number[];
+
+    constructor(auraType: AuraType, auraMultipliers: number[]) {
```

Comment:
You can write this line as
```js
constructor(
    public auraType: AuraType,
    public auraMultipliers: number[],
) { }
```
then you will not need the declarations above the constructor or the setting inside of the constructor, typescript will Just Know.